tests/test_challenges.py

```diff
@@ -0,0 +1,63 @@
+import ast
+import click
+import responses
+
+from click.testing import CliRunner
+from pylsy import pylsytable
+
+from evalai.challenges import challenges
+from tests.data import challenge_response
+
+
+class TestChallenges:
+
+    def setup(self):
+
+        json_data = ast.literal_eval(challenge_response.challenges)
+
+        responses.add(responses.GET, 'http://localhost:8000/api/challenges/challenge/all',
```

Comment:
Please don't hardcode the url. Try to look for some other way so that we will run it in prod. we don't have to modify it.